TunnelKit/Sources/Protocols/OpenVPN/AppExtension/NETCPLink.swift

```diff
@@ -72,7 +75,7 @@ class NETCPLink: LinkInterface {
                 var newBuffer = buffer
                 newBuffer.append(contentsOf: data)
                 var until = 0
-                let packets = PacketStream.packets(fromStream: newBuffer, until: &until)
+                let packets = PacketStream.packets(fromStream: newBuffer, until: &until, xorMask: self!.xorMask)
```

Comment:
Unwrapped self here is very likely to raise unpredictable, obscure exceptions on disconnection. Better add a check.